docker.local/bin/unit_test_build.sh

```diff
@@ -1,9 +1,26 @@
 #!/bin/bash
 set -e
 
+cmd="build"
+dockerfile="docker.local/build.unit_test/Dockerfile"
+platform=""
+
+for arg in "$@"
```

Comment:
What are these blocks added to the `docker.build.*.sh` files all about?

Reply:
Like everything in master branch, this is m1 support